Formula/rpki-client.rb

```diff
@@ -0,0 +1,55 @@
+class RpkiClient < Formula
+  desc "OpenBSD portable rpki-client"
+  homepage "https://www.rpki-client.org/index.html"
+  url "https://ftp.openbsd.org/pub/OpenBSD/rpki-client/rpki-client-7.4.tar.gz"
+  sha256 "efd336137707ad73fa51f8480cff686c21fea7086619432069c1a02553b68d57"
+  license "ISC"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on arch: :x86_64
+  depends_on "libressl"
+  depends_on macos: :mojave
+
+  # for test later on
+  resource("testdata") do
+    # use this to limit test to one RIR's TA and the minimum set of products
+    url "https://tal.apnic.net/apnic-as0.tal"
+    sha256 "0c5c2301a1da19b20f6a5407c10c121886d6298581e8112e2f8d500ae872e69b"
+  end
+
+  def install
+    ENV.deparallelize
```

Comment:
Does the build fail without this? If so, please report this upstream.

Reply:
tested, not needed, removed.